src/localization/locales/en-US.json

```diff
@@ -56,7 +58,20 @@
         "list_empty": "No Groups",
         "list_new_description": "Can’t find the organizations or groups you are affiliated with? Create a group for it.",
         "create": "Create a Group",
-        "list_new_button": "Create Group"
+        "list_new_button": "Create Group",
+        "members_title_manager": "Manage Members",
+        "members_description_manager": "Remove or invite members for your {{name}} to maintain the accureate member list in {{name}} profile page.",
+        "members_title_member": "{{name}} Members",
+        "members_description_member": "",
+        "members_list_title": "{{name}} Members",
+        "members_list_leave": "Leave",
+        "members_list_remove": "Remove",
+        "members_list_empty": "No members",
+        "members_list_column_name": "Name",
+        "members_list_column_role": "Role",
+        "members_remove_confirmation_title": "Remove Member",
+        "members_remove_confirmation_message": "You are about to remove “{{userName}}” from the “{{name}}” member list.",
```

Comment:
issue: `member list` is unclear. Would be good to make it clear they are being removed from a group.

Reply:
@paulschreiber Please sync with Hana and let me know the content that shoudl be displayed here

Reply:
@paulschreiber Sure
<img width="1266" alt="Screen Shot 2022-01-24 at 12 29 28 PM" src="https://user-images.githubusercontent.com/1316782/150833983-ea75e138-df82-4ba8-9aa9-b901ce7d7441.png">

Reply:
Please change to `"You are about to remove “{{userName}}” from the “{{name}}” group."`